src/pages/User/__snapshots__/User.test.tsx.snap

```diff
@@ -107,14 +107,19 @@ exports[`User Component tests > Account Page 1`] = `
         <div
           class="_panelHeader_e9c5ca"
         >
-          <h3>
+          <h2>
             account.about_you
-          </h3>
+          </h2>
         </div>
         <form
           class="_flexBox_1c1c63"
           novalidate=""
         >
+          <h1
+            class="hideUntilFocus"
+          >
+            nav.account
+          </h1>
```

Comment:
Also here, an `h1` after an `h2`. Is this correct?

Reply:
See my comment here: https://github.com/Videodock/ott-web-app/pull/3#discussion_r1430288079 😃

Reply:
Yes :)